tools/maian-analysis/signing_transactions.md

```diff
@@ -0,0 +1,26 @@
+### Overview
```

Comment:
What's the value here?  The analysis states that "Hedera does not support `eth_sendTransaction`".

Reply:
eth_sendRawTransaction can be used instead. But it would require us to rewrite part of the MAIAN tool code. We won't be able to use Hedera JSON RPC in MAIAN otherwise.

PR replace with: https://github.com/hashgraph/hedera-smart-contracts/pull/738